docs/ui/cloud-getting-started.md

```diff
@@ -79,13 +71,13 @@ Select the **+** button to create a new API key. You're prompted to provide a na
 
 Note that API keys cannot be revealed again in the UI after you generate them, so copy the key to a secure location.
 
-## Configure execution environment
+## Configure a local execution environment
 
-Now configure a local execution environment to use Prefect Cloud as the API server for flow runs.
+Configure a local execution environment to use Prefect Cloud as the API server for flow runs. In other words, "log in" to Prefect Cloud from a local environment where you want to run a flow.
 
 First, [Install Prefect](/getting-started/installation/) in the environment in which you want to execute flow runs.
 
-Next, use the Prefect CLI `prefect cloud login` command to log into Prefect Cloud from your environment, using the [API key](#create-an-api-key) generated previously.
+Next, use the `prefect cloud login` Prefect CLI command to log into Prefect Cloud from your environment, using the [API key](#create-an-api-key) generated previously.
```

Comment:
Should we have them generate an API key now that the CLI will do it for them?

Reply:
I would like to leave this in place until I update the service account login docs with examples. API keys remain an important way to configure an environment for a lot of production use cases.

Reply:
Great!